index.js

```diff
@@ -404,6 +511,19 @@ function setupYamlConfigs () {
     return fileConfig;
 }
 
+function startOrProgressVoteskip(channel) {
```

Comment:
Maybe also save the current song playing, there's an unhandled edge case here where the voteskip can be started at the end of one song and actually happen at the next song, thus including unintended voters.